gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-container/pom.xml

```diff
@@ -453,41 +437,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.xolstice.maven.plugins</groupId>
-                <artifactId>protobuf-maven-plugin</artifactId>
-                <version>0.6.1</version>
-                <configuration>
-                    <!--
-                      The version of protoc must match protobuf-java. If you don't depend on
-                      protobuf-java directly, you will be transitively depending on the
-                      protobuf-java version that grpc depends on.
-                    -->
-                    <protoTestSourceRoot>src/test/resources</protoTestSourceRoot>
-                    <protocArtifact>com.google.protobuf:protoc:${protobuf-java.version}:exe:${os.detected.classifier}
-                    </protocArtifact>
-                    <pluginId>grpc-java</pluginId>
-                    <pluginArtifact>io.grpc:protoc-gen-grpc-java:1.42.1:exe:${os.detected.classifier}</pluginArtifact>
-                    <protocPlugins>
-                        <protocPlugin>
-                            <id>vertx-grpc-protoc-plugin</id>
-                            <groupId>io.vertx</groupId>
-                            <artifactId>vertx-grpc-protoc-plugin</artifactId>
-                            <version>${vertx.version}</version>
-                            <mainClass>io.vertx.grpc.protoc.plugin.VertxGrpcGenerator</mainClass>
-                        </protocPlugin>
-                    </protocPlugins>
-                </configuration>
-                <executions>
-                    <execution>
-                        <id>compile</id>
-                        <goals>
-                            <goal>test-compile</goal>
-                            <goal>test-compile-custom</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
```

Comment:
We don't need this plugin anymore?

Reply:
No gRPC tests anymore here, I beleive they were moved to integration tests.
Thank you for mentioning it, turns out I  forgot to remove gRPC resources (proto &APIs) in the process...